### PR TITLE
MGMT-5255 Add events URL to CRD

### PR DESCRIPTION
**Pull request by rollandf**
Add DebugInfo struct under Status in AgentClusterInstall CRD.
Add EventsURL with a presigned URL to the events in REST API.

The URL will be cleared when the Cluster is deleted in the backend after the installation succeeded.

Signed-off-by: Fred Rolland <frolland@redhat.com>

### CODE REVIEW

**Comment by rollandf** [comment]
```
  debugInfo:
    eventsURL: http://10.1.37.42:6000/api/assisted-install/v1/clusters/9878074e-e503-4963-9138-81764faca01c/events?api_key=eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCJ9.eyJjbHVzdGVyX2lkIjoiOTg3ODA3NGUtZTUwMy00OTYzLTkxMzgtODE3NjRmYWNhMDFjIn0.UoKXXn8HrHrJcJaIKv9wOI2qdiMOc2l85dy7XRGFV9c0cNUKmR3DLSlJcb5CAO1EmC3GCzxlqTD3bWHvPQe5Ug
```

File: config/crd/bases/extensions.hive.openshift.io_agentclusterinstalls.yaml [hunk 1]
```diff
@@ -276,6 +276,15 @@ spec:
                 description: ControlPlaneAgentsDiscovered is the number of Agents
                   currently linked to this ClusterDeployment that are ready for use.
                 type: integer
+              debugInfo:
+                description: DebugInfo includes information for debugging installation
```
Comment (omertuc) [inline config/crd/bases/extensions.hive.openshift.io_agentclusterinstalls.yaml:280]: nit: `the installation`

File: config/crd/bases/extensions.hive.openshift.io_agentclusterinstalls.yaml [hunk 2]
```diff
@@ -276,6 +276,15 @@ spec:
...
+                  process.
+                properties:
+                  eventsURL:
+                    description: EventsURL specifies an HTTP/S URL that contains events
```
Comment (omertuc) [inline config/crd/bases/extensions.hive.openshift.io_agentclusterinstalls.yaml:284]: nit: `which occurred during the`

File: config/crd/bases/extensions.hive.openshift.io_agentclusterinstalls.yaml (hunk 1 above)
Comment (rollandf) [reply config/crd/bases/extensions.hive.openshift.io_agentclusterinstalls.yaml]: Done

File: config/crd/bases/extensions.hive.openshift.io_agentclusterinstalls.yaml (hunk 2 above)
Comment (rollandf) [reply config/crd/bases/extensions.hive.openshift.io_agentclusterinstalls.yaml:284]: Done

**Review by filanov — APPROVED** [review]

**Pull request merged**